axiom-init/src/projectScaffold/forge.ts

```diff
@@ -74,19 +71,34 @@ export const scaffoldForge = async (
 
   // Clone the axiom-quickstart template
   console.log("Fetching Axiom quickstart template...");
-  const tempDir = `.axiom-temp-${Date.now()}`; 
+  const tempDir = `.axiom-temp-${Date.now()}`;
   await sm.exec(`git clone --depth 1 https://github.com/axiom-crypto/axiom-quickstart.git ${tempDir}`, "Clone Axiom quickstart template");
 
-
-  // Delete the 'app' folder
-  await sm.exec(`rm -rf ${path.join(tempDir, "app")}`, "  - Process template");
+  // Delete .ts files in app folder (not recursive)
+  await sm.exec(`find ${path.join(tempDir, "app")} -maxdepth 1 -type f -name "*.ts" -delete`, "  - Process template");
 
   // Copy files to target path
   sm.cp(`${tempDir}/.`, ".", `  - Copy files to ${chalk.bold(sm.basePath)}`);
 
+  // Remove .git folder from cloned quickstart scaffold
+  await sm.rm(".git", `  - Remove .git folder from cloned quickstart scaffold`);
+
+  // Remove lib folder from cloned quickstart scaffold
+  await sm.rm("lib", `  - Remove lib folder from cloned quickstart scaffold`);
+
+  // Initialize git repo
+  await sm.exec("git init", "Initialize git repository");
+
```

Comment:
Let's make an initial commit here, so that the user doesn't need to manually add every file.

Reply:
sounds good